examples/with-supertokens/pages/index.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react'
 import Head from 'next/head'
 import styles from '../styles/Home.module.css'
-import ThirdPartyEmailPassword, {
-  ThirdPartyEmailPasswordAuth,
-} from 'supertokens-auth-react/recipe/thirdpartyemailpassword'
+import ThirdPartyEmailPassword from 'supertokens-auth-react/recipe/thirdpartyemailpassword'
 import supertokensNode from 'supertokens-node'
 import { backendConfig } from '../config/backendConfig'
```

Comment:
I think the version of this example app in the actual nextjs repo is in TS (someone else converted it to TS). Can you merge that into our fork and make these changes on top of the TS version?